etc/terraform.conf

```diff
@@ -30,7 +30,7 @@ HWE_X11="no"
 INCLUDE_APPCENTER=""
 
 # suffix for generated .iso files
-OUTPUT_SUFFIX=""
+OUTPUT_SUFFIX="rc"
```

Comment:
Is the RC suffic necessary now?

Reply:
Good question, I've asked it on Slack
https://ele.slack.com/archives/C8JPLKKQQ/p1676473060941199

Reply:
@tintou response: 
> No, we are not at RC stage anymore 

So I'll update this.